compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/text/selection/SelectionHandles.uikit.kt

```diff
@@ -52,6 +54,7 @@ private val RADIUS = 6.dp
  */
 private val THICKNESS = 2.dp
 
+@OptIn(ExperimentalComposeUiApi::class)
```

Comment:
BTW It's cross-module experimental, but that's not first usage of it. So I guess it's OK here.

Reply:
Thanks for notice about it!